api/python/quilt3/packages.py

```diff
@@ -1139,20 +1139,28 @@ def top_hash(self):
         """
```

Comment:
Please add the high-level top_hash algorithm to the docs here (i.e. note that metadata are hashed as of <commit>).

Reply:
> note that _get_top_hash_parts indicates what from each entry contributes to the hash

This docstring is public, why mention a private method here?

Reply:
Agreed to skip mention it on the docstring